deeppavlov_agent/utils/migrate_3.py

```diff
@@ -1,9 +1,9 @@
 import asyncio
 import argparse
 
-from core.state_manager import StateManager
-from core.state_schema import Dialog, Human, BotUtterance
-from core.db import DataBase
+from .core.state_manager import StateManager
+from .core.state_schema import Dialog, Human, BotUtterance
+from .core.db import DataBase
```

Comment:
Do these work?
i feel like instead of `.core` there should be `deeppavlov_agent.core`